google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryDryRunResult.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigquery;
+
+import java.util.List;
+
+public interface BigQueryDryRunResult {
```

Comment:
What's "dry run" in this context?

Reply:
It's a BQ thing where customers can run a "dryRun" where they don't get charged in order to get a sense of how much running this query could cost and also how this query looks like (i.e. results table schema, query parameters).